packages/client/hmi-client/src/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue

```diff
@@ -198,11 +231,19 @@ const selectedRunId = computed(
 	() => props.node.outputs.find((o) => o.id === selectedOutputId.value)?.value?.[0]
 );
 
+const parentSize = ref({ width: 0, height: 270 }); // Set the initial height or any default height
+// Calculate the parent container's width
+onMounted(async () => {
+	const parentContainerWidth = document.querySelector('#output-panel').clientWidth - 48;
+	parentSize.value.width = parentContainerWidth;
+});
```

Comment:
Do not identify elements by id, it doesn't always work. If you need to access HTML elements in Vue, use `ref`.

Also this function does not need to be `async`.

Reply:
fixed. thanks.